Editor/AGS.Editor/Components/ViewsComponent.cs

```diff
@@ -186,6 +192,11 @@ private void OnItemIDChanged(View item)
         {
             RePopulateTreeView(GetNodeIDForView(item));
             UpdateOpenWindowTitles();
+            Components.CharactersComponent cmp = ComponentController.Instance.FindComponent<Components.CharactersComponent>();

```

Comment:
Qualifying `Components` namespace isn't necessary and null check can be done with the `?.` operator

```suggestion
            ComponentController.Instance.FindComponent<CharactersComponent>()?.UpdateCharacterViews;
```